spec/hooks/useConsoleErrors/useConsoleErrors.test.tsx

```diff
@@ -21,7 +22,7 @@ describe('Testing Hook (client): useConsoleErrors', () => {
   });
 
   it('logs error if onAddToCartClick is missing', () => {
-    const quizOptions = { quizId: QUIZ_ID };
+    const quizOptions = { quizId: QUIZ_ID, callbacks: {} };
```

Comment:
```suggestion
    // @ts-expect-error Testing the error itself
    const quizOptions = { quizId: QUIZ_ID, callbacks: {} };
```

Reply:
Thoughts on disabling this locally instead of at the file level? (I couldn't think of a better comment, feel free to change it)

Reply:
👍 Makes sense to me. I did not know about/did not find the @ts-expect-error declaration.